table.cc

```diff
@@ -1085,6 +1085,10 @@ void table::set_metrics() {
     if (_config.enable_metrics_reporting) {
         _metrics.add_group("column_family", {
                 ms::make_derive("memtable_switch", ms::description("Number of times flush has resulted in the memtable being switched out"), _stats.memtable_switch_count)(cf)(ks),
+                ms::make_gauge("memtable_partition_writes", ms::description("Number of write operations performed on partitions in memtables"), [this] () { return _stats.memtable_partition_insertions + _stats.memtable_partition_hits; })(cf)(ks),
+                ms::make_gauge("memtable_partition_hits", ms::description("Number of times a write operation was issued on an existing partition in memtables"), _stats.memtable_partition_hits)(cf)(ks),
+                ms::make_gauge("memtable_row_writes", ms::description("Number of row writes performed in memtables"), [this] () { return _stats.memtable_row_insertions + _stats.memtable_row_hits; })(cf)(ks),
+                ms::make_gauge("memtable_row_hits", ms::description("Number of rows overwritten by write operations in memtables"), _stats.memtable_row_hits)(cf)(ks),
```

Comment:
These should be make_counter(). Gauges are for things that memory in use that can go up and down. Counters are for counted events that are intended to be viewed using the first derivative wrt. time.